src/kw_env.sh

```diff
@@ -124,6 +129,35 @@ function create_new_env()
   fi
 }
 
+# This Function gives the user the feature to destroy an environment.
+function destroy_env()
+{
+  local local_kw_configs="${PWD}/.kw"
+  local cache_build_path="$KW_CACHE_DIR"
+  local env_name=${options_values['DESTROY']}
+
+  if [[ ! -d "$local_kw_configs" ]]; then
+    complain 'It looks like that you did not setup kw in this repository.'
+    complain 'For the first setup, take a look at: kw init --help'
+    exit 22 # EINVAL
+  fi
+
+  if [[ ! -d "${local_kw_configs}/${env_name}" ]]; then
+    complain 'The environment' "${env_name}" 'does not exist.'
+    exit 22 # EINVAL
+  fi
+
+  if [[ $(ask_yN 'Are you sure you want to delete this environment?') =~ '0' ]]; then
+    exit 22 # EINVAL
+  fi
+
+  rm -rf "${local_kw_configs:?}/${env_name}" && rm -rf "${cache_build_path:?}/${env_name}"
+  success 'The' "$env_name" 'environment has been destroyed.'
+
+  # TODO: When the user deletes an environment he is using the symbolic links will be broken.
+  # A function is needed that returns to a "no env" state where the .kw links are not symbols.
```

Comment:
Do you already have that change that creates the no env option? Maybe you should include it as part of this commit and ensure that this feature is complete.

Reply:
I just created a new PR for this, I think it will be more organized.